test_vm/tests/power_scenario_tests.rs

```diff
@@ -85,3 +95,84 @@ fn create_miner_test() {
     };
     expect.matches(v.take_invocations().last().unwrap())
 }
+
+#[test]
+fn test_cron_tick() {
+    let store = MemoryBlockstore::new();
+    let mut vm = VM::new_with_singletons(&store);
+
+    let addrs = create_accounts(&vm, 1, BigInt::from(10_000u64) * BigInt::from(10u64.pow(18)));
+
+    // create a miner
+    let (id_addr, robust_addr) = create_miner(
+        &mut vm,
+        addrs[0],
+        addrs[0],
+        RegisteredPoStProof::StackedDRGWindow32GiBV1,
+        TokenAmount::from(10_000e18 as i128),
+    );
+
+    // create precommit
+    let seal_proof = RegisteredSealProof::StackedDRG32GiBV1; // p1??
+    let sector_number = 100;
+    let sealed_cid = make_sealed_cid(b"100");
+    let precommit_params = PreCommitSectorParams {
+        seal_proof,
+        sector_number,
+        sealed_cid,
+        seal_rand_epoch: vm.get_epoch() - 1,
+        deal_ids: vec![],
+        expiration: vm.get_epoch(), // todo
+        ..Default::default()
+    };
+    vm.apply_message(
+        addrs[0],
+        robust_addr,
+        TokenAmount::from(0u32),
+        MinerMethod::PreCommitSector as u64,
+        RawBytes::serialize(&precommit_params).unwrap(),
+    )
+    .unwrap();
```

Comment:
The problem is that you are serializing the params explicitly but the itest vm already does that for you.  So you are serializing a byte slice which doesn't match the precommit parameters.  To fix change line 133 to `precommit_params`.  The compiler will complain so then go toe PreCommitSectorParams declaration and add 
```
impl Cbor for PreCommitSectorParams {}
```

After you do this you will fail precommit because of the seal proof type. Fix that to be V1P1 and add in the expectation to call miner from power cron and you should pass

Reply:
You also need to address your todo on setting the precommit param expiration to keep this from failing

Reply:
Knocked it out, and tests are passing. Thanks for the look